### PR TITLE
Introduction to microservices part 4 draft.

### PR DESCRIPTION
**Pull request by sebadoom**
Draft. Complete text. Missing: diagrams, example, aside. RFC.
introduction to microservices part 4 dependencies and data sharing

### CODE REVIEW

**Comment by sebadoom** [comment]
Added diagrams.
introduction to microservices part 4 dependencies and data sharing

**Comment by chenkie** [comment]
* Break the first large paragraph out into a few paragraphs, at:
  * “In a traditional monolithic”
  * “This is what is commonly known as data sharing”
* In the text above the second diagram, change “product information” to “product catalog” just so the reader is clear that the diagram is used to illustrate the example given in the text
* Can we put a heading above “One important question you should ask yourself..” ? This will help to break up the text a bit
* Is there a definition of how much talking between microservices is too much?
* The “lots of talking” diagram might be more effective with **more** lines between the two. Maybe larger boxes with more lines?
* The event/subscription model diagram should have some way of communicating that a subscription is made. Maybe another box in between the represent events and then a line another line representing subscriptions? What do you think?

**Comment by sebadoom** [comment]
Complete article. RFC + ready for merging.
introduction to microservices part 4 dependencies and data sharing

File: _posts/2015-11-02-introduction-to-microservices-part-4-dependencies.markdown [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+- microservice
+- dependencies
+- data sharing
+- static data
+- data modelling
+- modelling
+- post-series
+---
+
+In this post in the microservices series we will study how to manage inter-service dependencies and how to deal with implicit interfaces in the form of data sharing. We will also extend our existing examples from the microservices series to show some of the concepts in this post.
```
Comment (mgonto) [inline _posts/2015-11-02-introduction-to-microservices-part-4-dependencies.markdown]: Link to the other posts here

**Comment by sebadoom** [comment]
Latest version.
introduction to microservices part 4 dependencies and data sharing

**Pull request merged**